.github/workflows/tests.yml

```
@@ -69,18 +92,29 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby-version: ["3.0", "3.1", "3.2", "3.3"]
+        ruby-version: ["3.1", "3.2", "3.3", "3.4"]
         rails-version:
-          - "6.1.7.6"
-          - "7.0.8"
-          - "7.1.1"
+          - "7.0"
+          - "7.1"
+          - "7.2"
+          - "8.0"
           - "main"
-        mysql-version: ["8.0", "8.2"]
+        mysql-version: ["8.0", "8.4"]
         exclude:
-          - ruby-version: "3.2"
-            rails-version: "6.1.7.6"
-          - ruby-version: "3.3"
-            rails-version: "6.1.7.6"
+          - ruby-version: "3.1"
+            mysql-version: "8.0"
+            rails-version: "main"
+          - ruby-version: "3.1"
+            mysql-version: "8.4"
+            rails-version: "main"
+
+          - ruby-version: "3.1"
+            mysql-version: "8.0"
+            rails-version: "8.0"
+          - ruby-version: "3.1"
+            mysql-version: "8.4"
+            rails-version: "8.0"
```

Same as above, I don't see what the benefit is of having the mysql versions in the exclusions matrix if it's not being used to make new combinations
```suggestion
          - ruby-version: "3.1"
            rails-version: "main"
          - ruby-version: "3.1"
            rails-version: "8.0"
```